src/db.py

```diff
@@ -323,9 +323,14 @@ def get_active_agents(self) -> Dict[str, AgentSpecSchema]:
     def get_core_config(self) -> Dict[str, str]:
         """Gets a list of configuration fields for the mquery core."""
         return {
+            # Autentication-related config
+            "auth_enabled": "Enable and force authentication for all users",
```

Comment:
Just making sure that this will be in fact str

Reply:
Yes. Right now there's no support for anything other than string configuration.

Everything has to end up serialised to string in the end. But I plan to add a layer of abstraction over the db config, since it's clearly getting a bit complicated. It will also help with validation.